.evergreen/generate_evergreen_tasks.js

```diff
@@ -378,6 +374,17 @@ OPERATING_SYSTEMS.forEach(
   }
 );
 
+BUILD_VARIANTS.push({
+  name: 'macos-1014',
```

Comment:
Just a suggestion to maybe alleviate some perf issues - maybe we try using `macos-1100` here to have the latest? There are 67 allocated hosts on macstadium for evergreen to use in this distro and maybe many other projects haven't updated to that yet?

Reply:
Sure, I'll bump it.